### ISSUE TITLE
Kriber fails to read monoclinic structures (Windows only)

### ISSUE THREAD

**Issue by stefsmeets** [stefsmeets/focus_package#1]
Error: (Pascal system I/O error 10 (file not found)

Kriber assumes the c-axis as the unique axis, so whenever it finds a structure with b-axis unique, it needs to apply an operation like `*P121  ==>P112(Y,Z,X)`, at which point it will crash. As a work around, redefine a monoclinic cell `A, B, C, 90, angle, 90`  as `B, C, A, 90, 90, angle`, and it will work.

In the `symdat` file, always the last one in a set of equivalent space groups will work. For example for `P2`, it should be defined as `P112` (c-axis unique).

**Comment by TroubleMaker1994** [comment]
Is the problem solved? I also encountered the same problem.

**Comment by stefsmeets** [comment]
No, I have looked at it from time to time, but I have a hard time understanding the code. I suggest you use the work-around, or run it on Linux if you have the possibility.
